lib/adapters/tool/index.ts

```diff
@@ -0,0 +1,12 @@
+import {TestplaneToolAdapter} from './testplane';
```

Comment:
Lets use dynamic import, and only import testplane tool adapter inside `case ToolName.Testplane`. User, which is using only playwright, will not have `testplane`, which is `./testplane/index.ts` importing

Reply:
Yeah, you are right. FIxed.